frontend/src/pages/org/archived-items.ts

```diff
@@ -266,45 +341,21 @@ export class CrawlsList extends LiteElement {
           </div>
         </header>
 
-        ${when(
-          this.archivedItems,
-          () => {
-            const { items, page, total, pageSize } = this.archivedItems!;
-            return html`
-              <section class="mx-2">
-                ${items.length
-                  ? this.renderArchivedItemList()
-                  : this.renderEmptyState()}
-              </section>
-              ${when(
-                total > pageSize,
-                () => html`
-                  <footer class="mt-6 flex justify-center">
-                    <btrix-pagination
-                      page=${page}
-                      totalCount=${total}
-                      size=${pageSize}
-                      @page-change=${async (e: PageChangeEvent) => {
-                        await this.fetchArchivedItems({
-                          page: e.detail.page,
-                        });
-
-                        // Scroll to top of list
-                        // TODO once deep-linking is implemented, scroll to top of pushstate
-                        this.scrollIntoView({ behavior: "smooth" });
-                      }}
-                    ></btrix-pagination>
-                  </footer>
-                `,
-              )}
-            `;
-          },
-          () => html`
+        ${this.archivedItemsTask.render({
+          initial: () => html`
             <div class="my-12 flex w-full items-center justify-center text-2xl">
               <sl-spinner></sl-spinner>
             </div>
           `,
-        )}
+          pending: () =>
+            // TODO differentiate between pending between poll and
```

Comment:
@emma-sg any ideas for a quick solve here? otherwise I'll track it in https://github.com/webrecorder/browsertrix/issues/1716